gpu_bdb/bdb_tools/q28_utils.py

```diff
@@ -60,18 +70,33 @@ def read_tables(config, c=None):
     return pr_df
 
 
-def gpu_hashing_vectorizer(x):
-    vec = HashingVectorizer(n_features=N_FEATURES,
-                            alternate_sign=alternate_sign,
-                            ngram_range=ngram_range,
-                            norm=norm,
-                            preprocessor=preprocessor
-     )
+def hashing_vectorizer(x):
+
+    if isinstance(x, cudf.Series):
+        vectorizer = HashingVectorizer
+        preprocessor = lambda s:s.str.lower()
```

Comment:
This is common for both so should be outside the if statement

Reply:
They need to use different hashing vectorizers (`HashingVectorizer` vs `SKHashingVectorizer`) and the lambda functions need to be slightly different (`s:s.str.lower()` vs `s:s.lower()`)